models/ens/ens_view_registrations.sql

```diff
@@ -2,12 +2,15 @@
         post_hook='{{ expose_spells(\'["ethereum"]\',
                                     "project",
                                     "ens",
-                                    \'["antonio-mendes","mewwts"]\') }}')}}
-SELECT *
+                                    \'["antonio-mendes","mewwts","steakhouse"]\') }}')}}
+SELECT *, NULL AS premium
 FROM {{source('ethereumnameservice_ethereum', 'ETHRegistrarController_1_evt_NameRegistered')}}
 UNION 
-SELECT *
+SELECT *, NULL AS premium
 FROM {{source('ethereumnameservice_ethereum', 'ETHRegistrarController_2_evt_NameRegistered')}}
 UNION 
-SELECT *
+SELECT *, NULL AS premium
```

Comment:
Hi @web3lyt, we can cast NULL to its expected type for `premium` field.

Reply:
That doesn't seem necessary. In sparksql, the datatype requirements are lax, and in dunesql, this still makes the field end up as a UINT256. Further, since this view needs to work in both sparksql and dunesql, I can't CAST the datatype to a common one unless I change it to INT in the raw table as well in both versions. It's much messier if I cast, this works better.